IlmBase/CMakeLists.txt

```diff
@@ -170,26 +170,10 @@ ENDIF ()
 INSTALL ( FILES
   ${CMAKE_CURRENT_BINARY_DIR}/config/IlmBaseConfig.h
   DESTINATION
-  include/OpenEXR
+  ${CMAKE_INSTALL_INCLUDEDIR}/OpenEXR
 )
 
-IF ( NOT WIN32 )
-  FILE ( WRITE ${CMAKE_BINARY_DIR}/IlmBase.pc "prefix=${CMAKE_INSTALL_PREFIX}\n" )
-  FILE ( APPEND ${CMAKE_BINARY_DIR}/IlmBase.pc "exec_prefix=\${prefix}
-libdir=\${exec_prefix}/lib
-includedir=\${prefix}/include
-OpenEXR_includedir=\${prefix}/include/OpenEXR
-
-Name: IlmBase
-Description: Base math and exception libraries
-Version: ${ILMBASE_VERSION}
-Libs: -L\${libdir} -lImath${ILMBASE_LIBSUFFIX} -lIexMath${ILMBASE_LIBSUFFIX} -lHalf${ILMBASE_LIBSUFFIX} -lIex${ILMBASE_LIBSUFFIX} -lIlmThread${ILMBASE_LIBSUFFIX} -pthread
-Cflags: -pthread -I\${OpenEXR_includedir}
-")
-
-  INSTALL ( FILES
-    ${CMAKE_BINARY_DIR}/IlmBase.pc
-    DESTINATION
-    lib/pkgconfig
-  )
-ENDIF()
+if ( PKG_CONFIG_FOUND )
```

Comment:
change this to `if ( NOT WIN32 )` or something. Making this conditional on whether you found `pkg-config` is brittle.